maplibre/src/context.rs

```diff
@@ -13,6 +13,8 @@ use crate::{
 /// Stores the camera configuration.
 pub struct ViewState {
     pub zoom: ChangeObserver<Zoom>,
+    pub maxzoom: u8,
```

Comment:
```suggestion
    pub maxZoom: u8,
```

Reply:
Also minzoom! But Rust requested, and now it is max_zoom
The symbols in all files mend to be identical? Also source.rs? There are a lot in style.rs!